src/Adyen/Service/Builder/Refund.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen API Library for PHP
+ *
+ * Copyright (c) 2020 Adyen N.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ *
+ */
+
+namespace Adyen\Service\Builder;
+
+use Adyen\Util\Currency;
+
+class Refund
+{
+    /**
+     * @param $orderAmount
+     * @param stirng $orderId
```

Comment:
Let's use phpdoc autogeneration otherwise typos like this is going to appear :)

```suggestion
     * @param string $orderId
```